packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.controller.js

```diff
@@ -2,13 +2,25 @@ import {
   USER_TYPE,
   PROOF_TYPE,
   TRACKING_TASK_TAG,
+  TRACKING_VARIABLES,
   LEGAL_LINK1,
   LEGAL_LINK2,
   LEGAL_LINK3,
   KYC_STATUS,
   DOCUMENTS_MATRIX,
 } from './user-identity-documents.constant';
 
+const replaceTrackingParams = (hit, params) => {
+  if (!params) return hit;
+  let formatted = hit;
+  params.forEach((param) => {
+    const { key, value } = Object.entries(param);
+    const replaced = hit.replace(`{${key}}`, value);
+    if (replaced) formatted = replaced;
+  });
+  return formatted;
+};
+
```

Comment:
Do we still need this ?
It seems it's used in the two tracking methods (`trackClick` and `trackPage`) which are never called with the `params` parameter meaning this method always exit on line 14 (`if (!params) return hit;`)